packages/alfa-style/src/node/predicate/is-transparent.ts

```diff
@@ -37,11 +40,12 @@ export function isTransparent(
           if (
             isText(node) &&
             isElement(parent) &&
-            Style.from(parent, device, context)
-              .computed("color")
-              .some(
-                (color) => color.type === "color" && Color.isTransparent(color)
-              )
+            hasComputedStyle(
+              "color",
+              Color.isTransparent,
+              device,
+              context
+            )(parent)
           ) {
             return true;
```

Comment:
The tests on `parent` can be grouped witha `and`.
The loop can likely be replaced by a `node.parent(Node.flatTree).some(…)`, probably extracting the `isText`.
I suspect that the two bits can then grouped together, maybe extracting a `hasTransparentParent` helper and then something like `or( and(isElement, hasComputedStyle), and(isText, hasTransparentParent))` 🤔